modules/uhttp/filters.go

```diff
@@ -68,11 +69,16 @@ func tracingServerFilter(host service.Host) FilterFunc {
 		if err != nil && err != opentracing.ErrSpanContextNotFound {
 			fxctx.Logger().Info("Malformed inbound tracing context: ", "error", err.Error())
 		}
-
 		span := opentracing.GlobalTracer().StartSpan(operationName, ext.RPCServerOption(spanCtx))
 		ext.HTTPUrl.Set(span, r.URL.String())
 		defer span.Finish()
 
+		jSpanCtx, ok := span.Context().(jaeger.SpanContext)
+		if ok {
+			fxctx.Logger().Info(
```

Comment:
I thought this task meant to add it to the logger (using With()), and not actually output the log, so that whenever user calls info or error, Logger would populate with these preset fields.